test/deployment.js

```diff
@@ -226,6 +226,13 @@ exports.test_valid_static_object_invalid_prop = () => {
 	assert.equal(isValid, false);
 };
 
+exports.test_project = () => {
+	const isValid = ajv.validate(deploymentConfigSchema, {
+		project: 'cool-project'
```

Comment:
is just `-`or `_` a valid name?

Reply:
For the project name we accept anything. It's just for displaying.